vips/vip-498/bsctestnet-2.ts

```diff
@@ -0,0 +1,257 @@
+const STALE_PERIOD_26H = 60 * 60 * 24 * 365; // 26 hours (pricefeeds with heartbeat of 24 hr)
+
+export const COMPTROLLER_CORE = "0x7Aa39ab4BcA897F403425C9C6FDbd0f882Be0D70";
+export const sUSDe = "0xA3A3e5ecEA56940a4Ae32d0927bfd8821DdA848A";
+export const USDe = "0x8bAe3E12870a002A0D4b6Eb0F0CBf91b29d9806F";
+
+export const VsUSDe_CORE = "0x33e4C9227b8Fca017739419119BbBA33A089D4a0";
+export const VUSDe_CORE = "0x36e8955c305aa48A99e4c953C9883989a7364a42";
+
+export const sUSDe_INITIAL_SUPPLY = parseUnits("10000", 18);
+export const USDe_INITIAL_SUPPLY = parseUnits("10000", 18);
+
+// Converters
+export const USDT_PRIME_CONVERTER = "0x3716C24EA86A67cAf890d7C9e4C4505cDDC2F8A2";
+export const USDC_PRIME_CONVERTER = "0x511a559a699cBd665546a1F75908f7E9454Bfc67";
+export const WBTC_PRIME_CONVERTER = "0x8a3937F27921e859db3FDA05729CbCea8cfd82AE";
+export const WETH_PRIME_CONVERTER = "0x274a834eFFA8D5479502dD6e78925Bc04ae82B46";
+export const XVS_VAULT_CONVERTER = "0xc203bfA9dCB0B5fEC510Db644A494Ff7f4968ed2";
+export const BaseAssets = [
+  "0x8d412FD0bc5d826615065B931171Eed10F5AF266", // USDT USDTPrimeConverter BaseAsset
+  "0x772d68929655ce7234C8C94256526ddA66Ef641E", // USDC USDCPrimeConverter BaseAsset
+  "0x92A2928f5634BEa89A195e7BeCF0f0FEEDAB885b", // WBTC WBTCPrimeConverter BaseAsset
+  "0x7b79995e5f793A07Bc00c21412e50Ecae098E7f9", // WETH WETHPrimeConverter BaseAsset
+  "0x66ebd019E86e0af5f228a0439EBB33f045CBe63E", // XVS XVSPrimeConverter BaseAsset
+];
+export const CONVERSION_INCENTIVE = parseUnits("3", 14);
+
+export const convertAmountToVTokens = (amount: BigNumber, exchangeRate: BigNumber) => {
+  const EXP_SCALE = parseUnits("1", 18);
+  return amount.mul(EXP_SCALE).div(exchangeRate);
+};
+
+const vip498 = () => {
+  const meta = {
+    version: "v2",
+    title: "[sepolia] New sUSDe and USDe market in the Core pool",
+    description: ``,
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+
+  return makeProposal(
+    [
+      // Add sUSDe Market
+      // oracle config
+      {
+        target: sepolia.REDSTONE_ORACLE,
+        signature: "setDirectPrice(address,uint256)",
+        params: [USDe, parseUnits("1", 18)],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.CHAINLINK_ORACLE,
+        signature: "setTokenConfig((address,address,uint256))",
+        params: [[USDe, CHAINLINK_USDe_FEED, STALE_PERIOD_26H]],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [USDe, [sepolia.REDSTONE_ORACLE, sepolia.CHAINLINK_ORACLE, sepolia.CHAINLINK_ORACLE], [true, true, true]],
+        ],
+        dstChainId: LzChainId.sepolia,
+      },
+
```

Comment:
We have to configure the thresholds in the BoundValidator for USDe. We usually set 0.99-1.01

Reply:
added BoundValidator